app/controllers/topics_controller.rb

```diff
@@ -893,36 +893,30 @@ def bulk
   end
 
   def reset_new
-    if params[:category_id].present?
-      category_ids = [params[:category_id]]
-      if params[:include_subcategories] == 'true'
-        category_ids = category_ids.concat(Category.where(parent_category_id: params[:category_id]).pluck(:id))
-      end
-
-      topic_scope =
-        if params[:tag_id]
-          Topic.where(category_id: category_ids).joins(:tags).where(tags: { name: params[:tag_id] })
-        else
-          Topic.where(category_id: category_ids)
```

Comment:
the topic scope is a filtered list to potential Dismiss. Depending on the combination of params like :category_id and :tag_id